message MetaInfo {
    required string msg_name = 1;
}

message CPUInfo {
    required string name = 1;
    required int32 usage_rate = 2;
}

message NetInfo {
    required string name = 1;
    required string ip = 2;
    required int64 recv_byte = 3;
    required int64 send_byte = 4;
}

message SimpleRequest {
    required string host_name = 1;
    required int64 datetime = 2;
    repeated CPUInfo cpu_infos = 3;
    repeated NetInfo net_infos = 4;
}

message SimpleResponse {
    required int32 return_code = 1;
    required string msg = 2;
}